src/diffusers/models/controlnet.py

```diff
@@ -24,19 +25,91 @@
 class ControlNetOutput(ControlNetOutput):
     def __init__(self, *args, **kwargs):
         deprecation_message = "Importing `ControlNetOutput` from `diffusers.models.controlnet` is deprecated and this will be removed in a future version. Please use `from diffusers.models.controlnets.controlnet import ControlNetOutput`, instead."
-        deprecate("ControlNetOutput", "0.34", deprecation_message)
+        deprecate("diffusers.models.controlnet.ControlNetOutput", "0.34", deprecation_message)
         super().__init__(*args, **kwargs)
 
 
 class ControlNetModel(ControlNetModel):
-    def __init__(self, *args, **kwargs):
+    def __init__(
```

Comment:
for models that inherits from `ConfigMixin`, it relies on its `__init__` method signature to identify expected keys and only pass the expected keys to the model https://github.com/huggingface/diffusers/blob/99c0483b67427de467f11aa35d54678fd36a7ea2/src/diffusers/configuration_utils.py#L465

Currently with its signature being `*args` and `**kwargs`, None of the configs are being passed to the `ControlNetModel` when importing from this legacy location.